src/driver/read_fixed.rs

```diff
@@ -1,27 +1,30 @@
 use crate::buf::fixed::FixedBuf;
-use crate::buf::{IoBuf, IoBufMut, Slice};
+use crate::buf::BoundedBufMut;
 use crate::driver::op::{self, Completable};
 use crate::driver::{Op, SharedFd};
 use crate::BufResult;
 
 use std::io;
 
-pub(crate) struct ReadFixed {
+pub(crate) struct ReadFixed<T> {
```

Comment:
Now it's generic over T, but before it doesn't have to be. It took a Slice<FixedBuf> buf. Wonder what the implications of that are.

Reply:
Now it will carry a `FixedBuf` if that's what the caller has passed, no need for a `Slice` wrapper in this case.